watson/watson.py

```diff
@@ -490,3 +489,48 @@ def report(self, from_, to, current=None, projects=None, tags=None,
 
         report['time'] = total.total_seconds()
         return report
+
+    def rename_project(self, old_project, new_project):
+        """Takes a project and renames it in all affected frames."""
```

Comment:
Docstring summary lines in imperative form, please.

    """Rename a project in all affected frames."""

Reply:
done